### ISSUE TITLE
Make a system by which unobtainable items are listed somewhere ingame

### ISSUE THREAD

**Issue by clusterfack** [d3athrow/vgstation13#9777]
The primary purpose of this issue is to point out the fact that unobtainable items, with neat features, litter our code but are totally unknown to everyone. This makes them totally wasteful, and I think admins should be able to find out which items are like this so that they can use them to spice up rounds and give them the use they deserve.

Now there's a few ways to do this 
- Make some kind of internal list of typepaths of unobtainable items, I honestly think this is pretty shitty and extremely difficult to maintain
- Make a flag for an items/mobs that are unobtainable in game, and apply it to them in the code, then populate a list of all these on roundstart by looping through all typesof() and checking this flag. I think this is the most feasible and maintainable way of doing this

(I also think we should being making more flag vars we have clearly already reached the limit of what our current single var flag system can do)

**Comment by Exxion** [comment]
Would this just include items that are supposed to always be unobtainable, or all items that are not currently obtainable? Or items that can be obtained, but only under certain very specific circumstances, too?

**Comment by Intigracy** [comment]
cow/rend is a fun one according to @Probe1

**Comment by Probe1** [comment]
there are many fun ones.

**Comment by clusterfack** [comment]
I doubt anybody has the patience to actually find all the objects in our code that are unobtainable but if people just tagged em when they saw em I think we'd be well on our way towards this.

**Comment by gbasood** [comment]
I think we should do this for things that are intended unobtainable, at least
Giving objects like this a flag makes it more obvious that they shouldn't throw a 'make me a vampire' item into a vault

**Comment by dylanstrategie** [comment]
We have a ton of unobtainable shit. Some WIP items, most admin guns (which are otherwise gloriously overpowered or fire dumb projectile types), items too broken to be kept in circulation, etc

I'm fine with whatever you try, but at the end of the day we need someone to be made aware of it. Someone who is going to recode them to be gud

**Comment by Probe1** [comment]
>fire dumb projectile types
gun that shoots bullets that turn into fire on impact 

- [ ] Yes
- [ ] Yes
- [ ] WHEN

**Comment by dylanstrategie** [comment]
By dumb projectile types I didn't even mean cool ones

Most projectile guns that are admin-only actually fire lasers

**Comment by Probe1** [comment]
Yea I know I'm just idea-guy meming.

**Comment by PJB3005** [comment]
> actually what am i even doing. incendiary rounds that set targets alight.

Traitor flare gun

**Comment by Probe1** [comment]
Yeah that's one, except its a flare gun that shoots shotgun shells.  I was thinking of a more traditional gun with special ammo.

Shieeeet I wonder if the Protolathe can even still handle reagents?  It still requires beakers to build but that'd be neat af to make a phosphorous magazine for an assault rifle.

**Comment by ComicIronic** [comment]
Protolathe does accept chems, the traitor flare gun also lets flare rounds embed and set people on fire, and you could try some kind of coverage tool to see what types are ever used with new().
